app/src/main-process/menu/build-default-menu.ts

```diff
@@ -434,12 +434,12 @@ export function buildDefaultMenu({
     },
   ]
 
-  if (!hasCurrentPullRequest && enableStartingPullRequests()) {
+  if (enableStartingPullRequests()) {
     branchSubmenu.push({
-      label: __DARWIN__ ? 'Start Pull Request' : 'Start pull request',
-      id: 'start-pull-request',
+      label: __DARWIN__ ? 'Preview Pull Request' : 'Preview pull request',
```

Comment:
After doing the suggested next actions changes, I realized that "Start Pull Request" doesn't exactly make sense to differentiate from creating a pull request. Decided to make consistent verbiage of "Preview Pull Request". But, I am completely open to discussion about what terminology would communicate the intention the best.